lib/message_builder.rb

```diff
@@ -122,4 +122,8 @@ def labels(pull_request)
       .map { |label| "[#{label['name']}]" }
       .join(' ')
   end
+
+  def html_encode(string)
```

Comment:
Is there a standard Ruby method which will do this for us? Seems like we might end up having to keep updating this method as we find more characters which cause problems.

Reply:
Could use something like `CGI::escapeHTML` but the Slack API docs say that only these three characters need escaping so I didn't want to require CGI.

Reply:
Ah okay, I suppose that seems fine then 👍